hack/ginkgo-integ.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
the amount of code duplication makes me a little sad. :(

Reply:
Do we really care about most of these arguments? Why not statically set them?

I plan to either modularize ginkgo-e2e.sh, or move some of this functionality into Go. But I'm not yet sure which is the better answer.

Reply:
We want to delete ginkgo-e2e.sh: https://github.com/kubernetes/test-infra/issues/3536

It is not obvious to me why we need this file as opposed to setting the right ginkgo flags?